src/Fixer/ControlStructure/YodaStyleFixer.php

```diff
@@ -114,6 +119,14 @@ protected function createConfigurationDefinition()
                 ->setAllowedTypes(['bool', 'null'])
                 ->setDefault(null)
                 ->getOption(),
+            (new FixerOptionBuilder('always_move_variable', 'Whether variables should always be on non assignable side when applying Yoda style.'))
+                ->setAllowedTypes(['bool'])
+                ->setDefault(false)
```

Comment:
By the way I would enable these options in our `.php_cs.dist` file and set their default value to `true` on 3.0 (or even drop the options). WDYT?

Reply:
I would like these new defaults, however I would not drop the options as it would stop SF from updating to 3.0 I think, but that is just me guessing.